crates/rolldown_utils/src/light_guess.rs

```diff
@@ -3,88 +3,59 @@ use mime::Mime;
 use phf::{phf_map, Map};
 use std::str::FromStr;
 
-pub static MIME_TYPES: Map<&'static str, &'static str> = phf_map! {
+use crate::mime::MimeExt;
+pub static MIME_TYPES: Map<&'static str, (&'static str, bool)> = phf_map! {
```

Comment:
How about use a struct instead of tuple here? It's kind of confusing about the `bool`.

Reply:
ok